src/main/scala/ch/epfl/bluebrain/nexus/admin/routes/OrganizationRoutes.scala

```diff
@@ -0,0 +1,109 @@
+package ch.epfl.bluebrain.nexus.admin.routes
+
+import akka.http.javadsl.server.Rejections.validationRejection
+import akka.http.scaladsl.model.StatusCodes
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.server.{Directive0, Route}
+import ch.epfl.bluebrain.nexus.admin.config.AppConfig.tracing.trace
+import ch.epfl.bluebrain.nexus.admin.directives.{AuthDirectives, QueryDirectives}
+import ch.epfl.bluebrain.nexus.admin.marshallers.instances._
+import ch.epfl.bluebrain.nexus.admin.organizations.{Organization, Organizations}
+import ch.epfl.bluebrain.nexus.admin.types.ResourceF.resourceMetaEncoder
+import ch.epfl.bluebrain.nexus.iam.client.IamClient
+import ch.epfl.bluebrain.nexus.iam.client.config.IamClientConfig
+import ch.epfl.bluebrain.nexus.iam.client.types.{AuthToken, Permission}
+import monix.eval.Task
+import monix.execution.Scheduler
+
+class OrganizationRoutes(organizations: Organizations[Task])(implicit iamClient: IamClient[Task],
+                                                             iamClientConfig: IamClientConfig,
+                                                             s: Scheduler)
+    extends AuthDirectives(iamClient)
+    with QueryDirectives
+    with CombinedRoutes {
+
+  private val read  = Permission.unsafe("organizations/read")
+  private val write = Permission.unsafe("organizations/write")
+
+  def routes: Route = combinedRoutesFor("orgs")
+
+  override def combined(implicit credentials: Option[AuthToken]): Route =
+    readRoutes ~ writeRoutes
+
+  private def readRoutes(implicit credentials: Option[AuthToken]): Route =
+    extractResourcePath { resource =>
+      (get & authorizeOn(resource, read)) { path =>
+        extractOrg(path) { name =>
+          parameter('rev.as[Long].?) {
+            case Some(rev) =>
```

Comment:
we could reduce this code by adding on Organizations.scala a method:
```scala
def fetch(organization: String, label: String, rev: Option[Long])
```
that just directs to the appropriate method